src/main/java/com/gistpetition/api/petition/application/PetitionService.java

```diff
@@ -64,6 +66,26 @@ public Long createPetition(PetitionRequest petitionRequest, Long userId) {
         return PetitionPreviewResponse.pageOf(petitionRepository.findAllByCategoryAndReleasedTrue(Category.of(categoryId), pageable));
     }
 
+    @Transactional(readOnly = true)
+    public Page<PetitionPreviewResponse> retrieveReleasedAndExpiredPetition(Pageable pageable) {
+        return PetitionPreviewResponse.pageOf(petitionRepository.findAllByCreatedAtBeforeAndReleasedTrue(LocalDateTime.now().minusDays(31), pageable));
+    }
+
+    @Transactional(readOnly = true)
+    public Page<PetitionPreviewResponse> retrieveReleasedAndExpiredPetitionByCategoryId(Long categoryId, Pageable pageable) {
+        return PetitionPreviewResponse.pageOf(petitionRepository.findAllByCategoryAndCreatedAtBeforeAndReleasedTrue(Category.of(categoryId), LocalDateTime.now().minusDays(31), pageable));
+    }
+
+    @Transactional(readOnly = true)
+    public Page<PetitionPreviewResponse> retrieveReleasedAndNotExpiredPetition(Pageable pageable) {
```

Comment:
```suggestion
    public Page<PetitionPreviewResponse> retrieveOngoingPetition(Pageable pageable) {
```

진행중인 청원이라는 비지니스적인 메시지를 던지면 좋을 것 같아요!

Reply:
네 동의합니다! ㅎㅎ